README.md

```diff
@@ -49,10 +49,8 @@ Basic usage for creating an IAM Policy granting full access to AWS Simple Storag
 
 ```hcl
 module "role-s3-full-access" {
-  source  = "mineiros-io/iam-policy/aws"
-  version = "~> 0.5.0"
-
-  name = "S3FullAccess"
+  source = "mineiros-io/iam-policy/aws"
+  version = "~> 0.5.2"
```

Comment:
name was removed and alignment is broken. pleas pre review PRs accordingly before marking them as ready for review.. ;)

Reply:
My bad, sorry